src/style.css

```diff
@@ -1 +1,102 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-weight: 300;
+}
+
+.main {
+  padding: 12px 0;
+  display: flex;
+  gap: 20px;
+  flex-direction: column;
+}
+
+.search {
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  background-color: #FFF;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)